src/server/services/__tests__/govuk-notify.test.ts

```diff
@@ -200,8 +202,8 @@ describe("GOVUK Notify service:", () => {
       );
     });
 
-    test("it returns false when sendEmail rejects", async () => {
-      const notifyClient = GovUKNotify.getNotifyClient();
+    test.skip("it returns false when sendEmail rejects", async () => {
```

Comment:
why did this need to be skipped?

Reply:
This is my bad. Since these tests use functions that have been moved out of the server folder I was planning to move these to the scheduler folder. But I wanted to do that after I fixed all the tests. I forgot to get back to them after doing that so I'll address this now.